### ISSUE TITLE
[PROGRESS] Ruchitraa

### ISSUE THREAD

**Issue by Ruchitraa** [Codezilla-Club/30-Days-of-Code#60]
## Participant Details
**Name/Nickname:** Ruchitraa
**Pronouns:** she/her
**Domain:** Machine Learning

I am a beginner and want to explore machine learning.
## Milestones
- [x] Code for 30 days

- [x] Learn new concepts in ML.
- [x] Do projects related to it using Python.
## Checklist
- [x] I've read the Code of Conduct, Rules and Guidelines
- [x] I've named this Issue as indicated
- [x] I've been assigned a mentor

**Comment by Ruchitraa** [comment]
Day 1

## What I Learned Today
- Object Oriented Programming in Python

## What I'll Learn Tomorrow
- Basic Markdown

**Comment by Ruchitraa** [comment]
Day 2

## What I Learned Today
- Basic Markdown

## What I'll Learn Tomorrow
- Setting up and using Anaconda Environment

**Comment by gokulk03** [comment]
just provide some more info in the participant details and also in the milestones

**Comment by Ruchitraa** [comment]
Day 3

## What I Learned Today
-  Setting up and using Anaconda Environment

## What I'll Learn Tomorrow
- Data Analytics with Python (youtube video)

**Comment by Ruchitraa** [comment]
Day 4

## What I Learned Today
- Data Analytics with Python (youtube video)

## What I'll Learn Tomorrow
- Start working on Week-1 Project.

**Comment by Ruchitraa** [comment]
Day 5

## What I Learned Today
-   Working on Week-1 Project.

## What I'll Learn Tomorrow
- Week 1 Project - Need to describe functions using Markdown.

**Comment by Ruchitraa** [comment]
Day 6

## What I Learned Today
-  Completed Week 1 Project

## What I'll Learn Tomorrow
- Numpy

**Comment by Ruchitraa** [comment]
Day 7

## What I Learned Today
- Numpy

## What I'll Learn Tomorrow
- Pandas

**Comment by Ruchitraa** [comment]
Day 8

## What I Learned Today
- Pandas

## What I'll Learn Tomorrow
- Matplotlib

**Comment by Ruchitraa** [comment]
Day 10

## What I Learned Today
- Matplotlib

## What I'll Learn Tomorrow
- YouTube video - Python Libraries for Data Science

**Comment by Ruchitraa** [comment]
Day 11

## What I Learned Today
- YouTube video - Python Libraries for Data Science

## What I'll Learn Tomorrow
- Scikit learn

**Comment by Ruchitraa** [comment]
Day 12

## What I Learned Today
- Scikit learn

## What I'll Learn Tomorrow
- Learn Encoding for Classification

**Comment by Ruchitraa** [comment]
Day 13

## What I Learned Today
- Learn Encoding for Classification

## What I'll Learn Tomorrow
- Plotting various graphs using Matplotlib

**Comment by Ruchitraa** [comment]
Day 14

## What I Learned Today
- Plotting various graphs using Matplotlib

## What I'll Learn Tomorrow
- A Guide To Complete Statistics For Data Science Beginners (from resources)

**Comment by Ruchitraa** [comment]
Day 15

## What I Learned Today
-  A Guide To Complete Statistics For Data Science Beginners (from resources)

## What I'll Learn Tomorrow
- An Extensive Step by Step Guide to Exploratory Data Analysis (from resources)

**Comment by Ruchitraa** [comment]
Day 16

## What I Learned Today
- An Extensive Step by Step Guide to Exploratory Data Analysis (from resources)

## What I'll Learn Tomorrow
- Perform Exploratory Data Analysis

**Comment by Ruchitraa** [comment]
Day 17

## What I Learned Today
- Perform Exploratory Data Analysis

## What I'll Learn Tomorrow
- Perform Exploratory Data Analysis (continue)

**Comment by Ruchitraa** [comment]
Day 18

## What I Learned Today
- Perform Exploratory Data Analysis (continue)

## What I'll Learn Tomorrow
- Understanding significance of Confusion Matrix and Box plots.

**Comment by Ruchitraa** [comment]
Day 19

## What I Learned Today
- Understanding significance of Confusion Matrix and Box plots.

## What I'll Learn Tomorrow
- Learn various metric and loss functions

**Comment by Ruchitraa** [comment]
Day 20

## What I Learned Today
- Learn various metric and loss functions

## What I'll Learn Tomorrow
- SONAR Rock vs Mine Prediction with Python (youtube)

**Comment by Ruchitraa** [comment]
Day 21

## What I Learned Today
- SONAR Rock vs Mine Prediction with Python (youtube)

## What I'll Learn Tomorrow
- XGBoost: Theory & Practice (from resources)

**Comment by Ruchitraa** [comment]
Day 22

## What I Learned Today
- XGBoost: Theory & Practice (from resources)

## What I'll Learn Tomorrow
- XGBoost: Theory & Practice (from resources)(continued)

**Comment by Ruchitraa** [comment]
Day 23

## What I Learned Today
- XGBoost: Theory & Practice

## What I'll Learn Tomorrow
- Youtube Video - Diabetes Prediction using machine learning with python

**Comment by Ruchitraa** [comment]
Day 24

## What I Learned Today
- Youtube Video - Diabetes Prediction using machine learning with python

## What I'll Learn Tomorrow
- continue the same

**Comment by Ruchitraa** [comment]
Day 25

## What I Learned Today
-  Diabetes Prediction using machine learning with python

## What I'll Learn Tomorrow
- Revise Numpy, Pandas, Matpotlib

**Comment by Ruchitraa** [comment]
Day 26

## What I Learned Today
-  Revise Numpy, Pandas, Matpotlib

## What I'll Learn Tomorrow
- Revise Confusion Matrix and box plots

**Comment by Ruchitraa** [comment]
Day 27

## What I Learned Today
- Revise Confusion Matrix and box plots

## What I'll Learn Tomorrow
- Learn more about Exploratory Data Analysis

**Comment by Ruchitraa** [comment]
Day 28

## What I Learned Today
- Learn more about Exploratory Data Analysis

## What I'll Learn Tomorrow
- continue the same

**Comment by Ruchitraa** [comment]
Day 29

## What I Learned Today
- Learn more about Exploratory Data Analysis

## What I'll Learn Tomorrow
- Youtube - House Price Prediction using machine learning with python

**Comment by Ruchitraa** [comment]
Day 30

## What I Learned Today
- Youtube - House Price Prediction using machine learning with python

## What I'll Learn Tomorrow
- Challenge Completed !!
